### PR TITLE
Move CoqworkmgrApi out of stm to it own library

### PR DESCRIPTION
**Pull request by SkySkimmer**
This lets coqworkmgr.exe drop the dependency on stm reducing its size from 29MB to 2.7MB.

### CODE REVIEW

**Review by gares — APPROVED** [review]

File: tools/coqworkmgr/dune [hunk 1]
```diff
@@ -0,0 +1,13 @@
+(library
+ (name coqworkmgrapi)
+ (public_name coq-core.coqworkmgrapi)
+ (modules coqworkmgrApi)
+ (wrapped false)
```
Comment (SkySkimmer) [inline tools/coqworkmgr/dune:5]: Considering the name of the module, wrapping seems useless.

**Comment by ppedrot** [comment]
Is somebody assigning? Otherwise I can do it.

**Review by ejgallego — APPROVED** [review]

**Comment by SkySkimmer** [comment]
assignee?

**Review by ppedrot — APPROVED** [review]

**Comment by ppedrot** [comment]
@coqbot merge now

**Comment by ppedrot** [comment]
No need for a full CI. @coqbot merge now

**Pull request merged**